records_mover/db/vertica/unloader.py

```diff
@@ -35,6 +35,9 @@ def temporary_loadable_directory_loc(self) -> Iterator[BaseDirectoryUrl]:
             with self.s3_temp_base_loc.temporary_directory() as temp_loc:
                 yield temp_loc
 
+    def has_temporary_loadable_directory_loc(self) -> bool:
+        raise NotImplementedError  # TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker